### PR TITLE
fix: cross chain dest amount

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- fix bug can not save tx bridge
- refactor function switchNetwork, fix bug when switch network on local.
- update ui import token 
image

### CODE REVIEW

File: src/hooks/bridge/useBridgeCallback.ts [hunk 1]
```diff
@@ -292,7 +294,15 @@ function useBridgeSwap(
...
             })
-            sendTxToKsSetting(chainId, chainIdOut, txHash, from_token, to_token, inputAmountStr, outputAmountStr ?? '')
+            sendTxToKsSetting(
+              chainId,
+              chainIdOut,
+              txHash,
+              from_token,
+              to_token,
+              inputAmountStr,
+              outputInfo?.outputAmount?.toString() ?? '',
```
Comment (nguyenhoaidanh) [inline src/hooks/bridge/useBridgeCallback.ts:304]: save db raw string

File: src/hooks/bridge/useBridgeCallback.ts [hunk 2]
```diff
@@ -178,7 +175,15 @@ function useRouterSwap(
...
```
Comment (nguyenhoaidanh) [inline src/hooks/bridge/useBridgeCallback.ts:185]: save db raw string

Comment (nguyenhuudungz) [reply src/hooks/bridge/useBridgeCallback.ts:185]: Please use `outputAmountStr`.

File: src/hooks/bridge/useBridgeCallback.ts (hunk 1 above)
Comment (nguyenhuudungz) [reply src/hooks/bridge/useBridgeCallback.ts:304]: Please use `outputAmountStr`.

**Review by nguyenhuudungz — CHANGES_REQUESTED** [review]

File: src/hooks/bridge/useBridgeCallback.ts (hunk 2 above)
Comment (nguyenhuudungz) [reply src/hooks/bridge/useBridgeCallback.ts:185]: No need save raw string, save formatted string pls.

File: src/hooks/bridge/useBridgeCallback.ts (hunk 1 above)
Comment (nguyenhuudungz) [reply src/hooks/bridge/useBridgeCallback.ts:304]: No need save raw string, save formatted string pls.

File: src/hooks/bridge/useBridgeCallback.ts (hunk 2 above)
Comment (nguyenhoaidanh) [reply src/hooks/bridge/useBridgeCallback.ts:185]: whyyyy.
save raw string, in the later we can format any way we want

File: src/hooks/bridge/useBridgeCallback.ts (hunk 1 above)
Comment (nguyenhuudungz) [reply src/hooks/bridge/useBridgeCallback.ts:304]: Rounded to 6 digits is totally fine.

File: src/hooks/bridge/useBridgeCallback.ts (hunk 2 above)
Comment (nguyenhoaidanh) [reply src/hooks/bridge/useBridgeCallback.ts:185]: image
I update format here. 
In the future we want to format 10 digit, we don't have to do anything

Comment (nguyenhuudungz) [reply src/hooks/bridge/useBridgeCallback.ts:185]: Okay.

**Review by nguyenhuudungz — APPROVED** [review]

**Review by nguyenhuudungz — APPROVED** [review]

**Pull request merged**